tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "noEmit": true,
     "allowJs": true,
     "checkJs": true,
-    "target": "esnext",
+    "target": "es2019",
```

Comment:
Please revert. Recommend against changing any repository's settings in a PR.

Reply:
```suggestion
    "target": "esnext",
```